File: bot-api/src/main/java/gg/mineral/bot/api/instance/Session.java
package gg.mineral.bot.api.instance;

public interface Session {

    /**
     * Gets the player's access token.
     * 
     * @return the player's access token
     */
    String getAccessToken();

    /**
     * Gets the player's session ID.
     * 
     * @return the player's session ID
     */
    String getSessionId();
}

File: bot-api/src/main/kotlin/gg/mineral/bot/api/instance/Session.kt
package gg.mineral.bot.api.instance

interface Session {
    /**
     * Gets the player's access token.
     *
     * @return the player's access token
     */
    val accessToken: String

    /**
     * Gets the player's session ID.
     *
     * @return the player's session ID
     */
    val sessionId: String

    /**
     * Gets the player's username.
     *
     * @return the player's username
     */
    val username: String
}
